example/.flutter-plugins-dependencies

```diff
@@ -0,0 +1 @@
+{"_info":"// This is a generated file; do not edit or check into version control.","dependencyGraph":[{"name":"flutter_vlc_player","dependencies":[]}]}
```

Comment:
It looks like this file shouldn't be a part of version control

Reply:
Good call, I fill fix. First time doing Flutter. I should set up a flutter gitignore.